velox/dwio/parquet/reader/PageReader.h

```diff
@@ -446,6 +492,12 @@ class PageReader {
   // return to the caller.
   dwio::common::BitConcatenation nullConcatenation_;
 
+  // These three parameters are used to estimate the size of Parquet Page
+  // Header.
+  const uint32_t kPageHeaderBufferSize_ = 512;
```

Comment:
`k` prefix is reserved for compile time constants.  Just the normal `pageHeaderBufferSize_` is good.  Also `sizeof(PageHeader)` is a good default here.